docs/pre-release/howtos/cluster-in-vm.md

```diff
@@ -0,0 +1,192 @@
+---
+description: "Use Telepresence to intercept services in a cluster running in a hosted virtual machine."
+---
+
+# Network considerations for locally hosted clusters
+
+## The problem
+Telepresence creates a Virtual Network Interface ([VIF](../tun-device)) that maps the clusters subnets to the host machine when it connects. This is not a problem under normal circumstances (when the cluster runs in the cloud), but network problems may arise when the cluster runs in a VM that is hosted on the same machine where Telepresence runs.
```

Comment:
On the second sentence, I'd reword:

"If you're running Kubernetes locally (e.g., k3s, Minikube, Docker for Desktop), you may encounter network problems ..."